### ISSUE TITLE
wk1 jul_epic BUG: Facility File - Reset Pivot Chart Button crashes Excel 

### ISSUE THREAD

**Issue by lijenn** [lzim/teampsd#2116]
Discussed at Thurs. 7/7 Quant WG at 11am Pacific:

@lijenn Was able to propagate a facility file on Tuesday 7/6 following the propagation documentation Ash provided, but the Reset Pivot Chart button crashes and auto-closes the entire excel file.
No error message/code is produced.

@anazariz will look into the issue as the issue was not present in facility files he propagated on Thursday 7/1.
May be a Cloud issue than a code issue.

Please provide an estimate once you're able to.

**Comment by anazariz** [comment]
I tested a facility off of the cloud. All 4 viz Tab resets worked fine.
Then, I tested the same facility in the cloud. The reset on all 4 tabs worked fine. 

If there is a problem, it's bandwidth/cloud related.

**Comment by anazariz** [comment]
The load on this 32 bit Excel file is huge. This screenshot shows the load when I click on the Reset button.
One way to avoid this would be to break up the file into 4 components, HF, Meas, Diag and Enc. 
The drawback with that is that code is spaghetti style. Separating it out must be done carefully and will be time consuming.
 
![image.png](https://images.zenhubusercontent.com/5e4c0724779eee67006fe6c6/fabbc529-4976-4fd5-a2c7-bbc66236fdef)

**Comment by staceypark** [comment]
@anazariz When you say "break up" do you mean in the back end or the front end? We cannot have 4 separate excel files for the learners

**Comment by anazariz** [comment]
I re-propagated the Facility, which @staceypark and I talked about on 7/8 (the one that had problems with Excel aborting when the vizHF reset button was pressed). 

I then tested (using the internet explorer) changing the HealthFactorCategory filter multiple times and then resetting it. I ran the test twice (Exited the Excel workbook and restarted it). Each time I tested the reset press multiple times.  It worked fine for me. 
I also looked at the load on my machine after the reset had finished (to establish a baseline). Although using Task Manager is the crudest to measure Excel's performance, it gives a baseline of what the machine is doing in general. The baseline activities can  be contributed to software other Excel (e.g. Teams, etc.), which leaves Excel less resources to work with.       

![image](https://user-images.githubusercontent.com/39805164/125115391-20866580-e0a0-11eb-9618-962f786e1e70.png)
